rioting-daisies/BangazonWorkforce
# Finished details for department with list of employees

# Description

Details button on department shows that departments Name, budget, and a list of its employees with their first and last names displayed



Fixes Issue # https://github.com/rioting-daisies/BangazonWorkforce/issues/6    and https://github.com/rioting-daisies/BangazonWorkforce/issues/13

## Type of change

Please delete options that are not relevant.




# Testing Instructions

Git fetch --all and checkout cm-department-details
Run bangazon and click on the departments tab then click the details on a department with employees
Make sure the departments name budget and all its employees names are displayed
have fun!!



# Checklist:
